adapters/AppLovin/AppLovinAdapter/GADMAppLovinExtraAssets.m

```diff
@@ -0,0 +1,12 @@
+//
+//  GADMAppLovinExtraAssets.m
+//  SDK Network Adapters Test App
+//
+//  Created by Santosh Bagadi on 4/11/18.
+//  Copyright © 2018 AppLovin Corp. All rights reserved.
+//
+
+#import "GADMAppLovinExtraAssets.h"
+
+NSString *const GADMAppLovinAdID = @"ad_id";
+NSString *const GADMAppLovinCaption = @"caption";
```

Comment:
Is there a reason why these keys are in a separate class on iOS, vs Android being in the adapter? I have no preference, but:

1. If you were to do it here, it would be a cleaner design to user class properties.
2. If